db/migrate/20210411224620_add_user_id_to_companies.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdToCompanies < ActiveRecord::Migration[6.1]
+  def change
+    add_column :companies, :user_id, :integer
+    add_index :companies, :user_id
```

Comment:
have a look at these ones:

```
add_reference :companies, :user, index: true
add_foreign_key :companies, :users
```

This would do a lot of other things for us.

Reply:
You can look up these method names to see the difference between these ones and add_column + add_index